### PR TITLE
Eh aws marketplace

### PR DESCRIPTION
**Pull request by VioletteLepercq**

### CODE REVIEW

File: enterprise_hub_aws_marketplace.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+title: "The Enterprise Hub on the AWS Marketplace: Pay with your AWS Account"
+thumbnail: /blog/assets/158_aws_marketplace/thumbnail.jpg
+authors:
+- user: Violette
+- user: philschmid
```
Comment (philschmid) [inline enterprise_hub_aws_marketplace.md:6]:
```suggestion
```

File: assets/158_aws_marketplace/01_bis_offering.jpg
Comment (philschmid) [inline assets/158_aws_marketplace/01_bis_offering.jpg:1]: please upload the images to https://huggingface.co/datasets/huggingface/documentation-images/tree/main/blog

Comment (VioletteLepercq) [reply assets/158_aws_marketplace/01_bis_offering.jpg:1]: done

Comment (philschmid) [reply assets/158_aws_marketplace/01_bis_offering.jpg:1]: all images

Comment (VioletteLepercq) [reply assets/158_aws_marketplace/01_bis_offering.jpg:1]: even the thumbnail ?

File: enterprise_hub_aws_marketplace.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+thumbnail: /blog/assets/158_aws_marketplace/thumbnail.jpg
+authors:
+- user: Violette
+- user: sbrandeis
+- user: jeffboudier
+---
+
+# The Enterprise Hub on the AWS Marketplace: Pay with your AWS Account.
+
+AWS Enterprises users can now use Hugging Face as a safe & private collaboration platform without sharing payment information with Hugging Face.
```
Comment (julien-c) [inline enterprise_hub_aws_marketplace.md:12]:
```suggestion
AWS Enterprise users can now use Hugging Face as a safe & private collaboration platform with all billing handled directly on AWS.
```

**Review by julien-c — COMMENTED** [review]
what are our success metrics on this, ie. in 2 months what kind of % of Enterprise Hub do we see coming from this? (we can discuss internally 😁)

File: assets/158_aws_marketplace/01_bis_offering.jpg
Comment (philschmid) [reply assets/158_aws_marketplace/01_bis_offering.jpg:1]: no thumbnail is fine thats needed.

File: enterprise_hub_aws_marketplace.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
+---
+title: "The Enterprise Hub on the AWS Marketplace: Pay with your AWS Account"
```
Comment (jeffboudier) [inline enterprise_hub_aws_marketplace.md:2]:
```suggestion
title: "Subscribe to Enterprise Hub with your AWS Account"
```

File: enterprise_hub_aws_marketplace.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+thumbnail: /blog/assets/158_aws_marketplace/thumbnail.jpg
+authors:
+- user: Violette
+- user: sbrandeis
+- user: jeffboudier
+---
+
+# The Enterprise Hub on the AWS Marketplace: Pay with your AWS Account.
```
Comment (jeffboudier) [inline enterprise_hub_aws_marketplace.md:10]:
```suggestion
# Subscribe to Enterprise Hub with your AWS Account
```

File: enterprise_hub_aws_marketplace.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+AWS Enterprise users can now use Hugging Face as a safe & private collaboration platform with all billing handled directly on AWS.
```
Comment (jeffboudier) [inline enterprise_hub_aws_marketplace.md:12]:
```suggestion
You can now upgrade your Hugging Face Organization to Enterprise using your AWS account - get started [on the AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-n6vsyhdjkfng2).
```

File: enterprise_hub_aws_marketplace.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+## What is the Enterprise Hub, the famous 🤗 Hub but in Enterprise mode?
```
Comment (jeffboudier) [inline enterprise_hub_aws_marketplace.md:14]:
```suggestion
## What is Enterprise Hub? 
```

File: enterprise_hub_aws_marketplace.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+- user: jeffboudier
+---
+
+# Subscribe to Enterprise Hub with your AWS Account
+
+You can now upgrade your Hugging Face Organization to Enterprise using your AWS account - get started [on the AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-n6vsyhdjkfng2).
+
+## What is Enterprise Hub? 
+
+In addition to offering widely adopted open-source libraries and thousands of models and datasets, we enable an enterprise-ready version of the world’s leading AI platform. With the Enterprise Hub, you give your team the most advanced GDPR & SOC2T2 compliant platform to build AI with open-source resources complying with privacy and security Enterprise constraints:
```
Comment (jeffboudier) [inline enterprise_hub_aws_marketplace.md:16]:
```suggestion
Enterprise Hub is a premium subscription to upgrade a free Hugging Face organization with advanced security features, access controls, collaboration tools and compute options. With Enterprise Hub, companies can build AI privately and securely within our GDPR compliant and SOC2 Type 2 certified platform. Exclusive features include: 
```

File: enterprise_hub_aws_marketplace.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+
+## What is Enterprise Hub? 
+
+In addition to offering widely adopted open-source libraries and thousands of models and datasets, we enable an enterprise-ready version of the world’s leading AI platform. With the Enterprise Hub, you give your team the most advanced GDPR & SOC2T2 compliant platform to build AI with open-source resources complying with privacy and security Enterprise constraints: 
+- Benefit from state-of-the-art machine learning, 
+- Run inference workloads, 
+- Compute budget for your team, 
+- Choose the location of your Hugging Face repos/storage, 
+- Use your company's SSO with hf.co and increase security & productivity
+- Enable teams in regulated environments to keep up with the pace of open-source advancement frictionlessly
```
Comment (jeffboudier) [inline enterprise_hub_aws_marketplace.md:22]:
```suggestion
- Single Sign-On: ensure all members of your organization are employees of your company.
- Resource Groups: manage teams and projects with granular access controls for repositories.
- Storage Regions: store company repositories in Europe for GDPR compliance.
- Audit Logs: access detailed logs of changes to your organization and repositories.
- Advanced Compute Options: give your team higher quota and access to more powerful GPUs.
- Private Datasets Viewer: enable the Dataset Viewer on your private datasets for easier collaboration.
- Train on DGX Cloud: train LLMs without code on NVIDIA H100 GPUs managed by NVIDIA DGX Cloud.
- Premium Support: get the most out of Enterprise Hub and control your costs with dedicated support.
```

File: enterprise_hub_aws_marketplace.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+
+How can I enable the Enterprise Hub with my AWS account?
```
Comment (jeffboudier) [inline enterprise_hub_aws_marketplace.md:24]:
```suggestion
If you're admin of your organization, you can upgrade it easily with a credit card. But how do you upgrade your organization to Enterprise Hub using your AWS account? We'll walk you through it step by step below.
```

File: enterprise_hub_aws_marketplace.md [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+![Subscribe Now](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/08_subscribe.jpg "Subscribe Now")
+
+Now select the number of Enterprise Hub seats you are willing to buy for your organization, the billing frequency and click on Checkout. 
+
+![Select Seats](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/09_select.jpg "Select Seats")
+
+
+### Congratulations! 🥳
+
+Your organization is now subscribed to the Enterprise Hub, and billing is directly managed by your AWS account. All members of your Enterprise Hub can now start using safely and privately the Hugging Face Hub.
```
Comment (jeffboudier) [inline enterprise_hub_aws_marketplace.md:83]:
```suggestion
Your organization is now upgraded to Enterprise Hub, and its billing is directly managed by your AWS account. All members of your organization can now benefit from the advanced features of Enterprise Hub to build AI privately and securely.
```

**Review by jeffboudier — APPROVED** [review]
Added a few more changes. I think either the content of this blog post should replace the existing blog post https://huggingface.co/blog/aws-marketplace, or we should update the this existing post with up to date screenshots / content as the name of the AWS listing has changed

**Review by jeffboudier — APPROVED** [review]
LGTM!

File: assets/158_aws_marketplace/01_bis_offering.jpg
Comment (julien-c) [inline assets/158_aws_marketplace/01_bis_offering.jpg:1]: neeed to move those assets somewhere else before merging

File: enterprise_hub_aws_marketplace.md
Comment (julien-c) [inline enterprise_hub_aws_marketplace.md:1]: use dashes instead of underscores for consistency

File: assets/158_aws_marketplace/01_bis_offering.jpg
Comment (VioletteLepercq) [reply assets/158_aws_marketplace/01_bis_offering.jpg:1]: the blog is not referring to this asset but this one https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/01_bis_offering.jpg

File: enterprise_hub_aws_marketplace.md
Comment (VioletteLepercq) [reply enterprise_hub_aws_marketplace.md:1]: done now it is titled "enterprise-hub-aws-marketplace"

**Review by VioletteLepercq — COMMENTED** [review]
ready to be merged

File: assets/158_aws_marketplace/01_bis_offering.jpg
Comment (VioletteLepercq) [reply assets/158_aws_marketplace/01_bis_offering.jpg:1]: I dot understand what I need to do there - can you help ?

Comment (julien-c) [reply assets/158_aws_marketplace/01_bis_offering.jpg:1]: yes, you just need to delete them from this branch (so they're not merged in the repo)

Comment (VioletteLepercq) [reply assets/158_aws_marketplace/01_bis_offering.jpg:1]: ok done

**Review by julien-c — APPROVED** [review]
lgtm

**Pull request merged**